gradle-plugin/doc/src/main/asciidoc/inc/tasks/_tasks.adoc

```diff
@@ -0,0 +1,28 @@
+
+[[tasks]]
+= Tasks Overview
+
+This plugin supports a rich set for providing a smooth Java developer experience. These tasks can be categorized in multiple groups:
+
+* <<tasks-build, Build tasks>> are all about creating and managing Kubernetes build artifacts like Docker images or S2I builds.
+
+* <<tasks-develop, Development tasks>> target help not only in developing resource descriptors to the development cluster but also to manage the lifecycle of the development cluster as well.
+
+[[tasks-build]]
+.Build Tasks
```

Comment:
Not sure if build tasks is appropriate enough, since these include deployment tasks too, right?